lib/howl/config.moon

```diff
@@ -140,25 +162,44 @@ watch = (name, callback) ->
   append list, callback
 
 proxy_mt = {
-  __index: (proxy, key) ->
-    base = rawget proxy, '_base'
-    if base then base[key] else get key
-  __newindex: (proxy, key, value) -> proxy_set key, value, proxy
+  __index: (proxy, key) -> get(key, proxy._scope, proxy._read_layer)
+  __newindex: (proxy, key, value) -> set(key, value, proxy._scope, proxy._write_layer)
 }
 
-local_proxy = ->
-  proxy = {}
-  proxy.chain_to = (base) -> rawset proxy, '_base', base
+proxy = (scope, write_layer='default', read_layer=write_layer) ->
+  proxy = {
+    clear: => scopes[@_scope] = {}
+    _scope: scope
+    _write_layer: write_layer
+    _read_layer: read_layer
+  }
   setmetatable proxy, proxy_mt
 
+
+copy = (scope, new_scope) ->
```

Comment:
I'm missing a spec for this (ditto for `delete` and `move`). I'm missing it particularly much for this one as it doesn't seem right - this is not a copy operation but an aliasing operation. I'll comment further where it's actually used in Buffer.